src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -428,6 +453,11 @@ public boolean clearOfIntakeZone() {
     }
   }
 
+  /** Returns true if the lift is scoring in a high location */
+  public boolean isScoringHigh(){
+    return scoreLoc.getScoreHeight() == ScoreHeight.HIGH ? true : false;
+  }
```

Comment:
Move to scoring util

Reply:
Also can just return `scoreLoc.getScoreHeight() == ScoreHeight.HIGH` instead of the ternary